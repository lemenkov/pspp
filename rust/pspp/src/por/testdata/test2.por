ASCII SPSS PORT FILE                    ASCII SPSS PORT FILE                    
ASCII SPSS PORT FILE                    ASCII SPSS PORT FILE                    
ASCII SPSS PORT FILE                    0000000000000000000000000000000000000000
0000000000000000000000000123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrst
uvwxyz .<(+0&[]!$*);^-/|,%_>?`:#@'="000000~000000000000000000000{}\0000000000000
00000000000000000000000000000000000000000000000000000000SPSSPORTA8/200409316/   
   1D/STAT/TRANSFER44/5A/70/1/C5/9/2/5/9/2/CB/ALCOHOL (C)70/2/SE5/9/2/5/9/2/CG/S
ELF-ESTEEM (SE)70/3/BAC5/9/2/5/9/2/CR/BLOOD ALCOHOL CONTENT (BAC)70/5/SELFD5/9/2
/5/9/2/CF/SELF-DISCLOSUREF0/3.3/4.6/3.L/0/2.F/1.F/5.L/0/3/2/4.C/0/2/2.6/3.6/0/3.
6/1/6.R/0/1.O/1.L/3.F/0/2.9/2.L/6.C/0/2.C/2.I/5.O/0/1/2.O/4.C/0/3.6/3/5.O/0/1/3.
L/3.O/0/4.L/3.9/5.I/0/2.L/2.R/5/0/4.L/3.L/7/0/1.O/2.9/4/0/2.6/3.F/3.I/0/3.I/2.6/
5.R/0/3.R/3.C/5/0/3.L/1.C/5.L/0/2.R/2.R/3.C/1/2.C/5.9/5.9/1/1.L/5.C/5.R/1/4.3/6.
6/5.3/1/3.I/4.F/5.R/1/5/6.R/5/1/2.L/2.9/6.R/1/3.I/5.I/6.9/1/2.I/5.F/3.R/1/3.F/4.
F/5.R/1/3.9/3/5.R/1/3.C/5.3/4.O/1/2.O/4/4.O/1/2.3/4.I/5.O/1/3.9/6.L/6.6/1/3.9/4.
R/7.F/1/3/5.F/5.F/1/3.F/4.I/7/1/4.F/5.L/6.C/1/2.I/6.6/6/1/4.I/5.3/4.F/ZZZZZZZZZZ