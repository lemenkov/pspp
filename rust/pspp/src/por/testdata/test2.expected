Invalid date 20040931.

╭───────┬─────────────╮
│Product│STAT/TRANSFER│
╰───────┴─────────────╯

╭─────────┬─╮
│Variables│4│
╰─────────┴─╯

                                                                     Variables
╭───────────────────────────┬────────┬───────────────────────────┬─────────────────┬─────┬─────┬─────────┬────────────┬────────────┬──────────────╮
│                           │Position│           Label           │Measurement Level│ Role│Width│Alignment│Print Format│Write Format│Missing Values│
├───────────────────────────┼────────┼───────────────────────────┼─────────────────┼─────┼─────┼─────────┼────────────┼────────────┼──────────────┤
│ALCOHOL (C)                │       1│ALCOHOL (C)                │                 │Input│    8│Right    │F9.2        │F9.2        │              │
│SELF-ESTEEM (SE)           │       2│SELF-ESTEEM (SE)           │                 │Input│    8│Right    │F9.2        │F9.2        │              │
│BLOOD ALCOHOL CONTENT (BAC)│       3│BLOOD ALCOHOL CONTENT (BAC)│                 │Input│    8│Right    │F9.2        │F9.2        │              │
│SELF-DISCLOSURE            │       4│SELF-DISCLOSURE            │                 │Input│    8│Right    │F9.2        │F9.2        │              │
╰───────────────────────────┴────────┴───────────────────────────┴─────────────────┴─────┴─────┴─────────┴────────────┴────────────┴──────────────╯

╭────┬───────────┬────────────────┬───────────────────────────┬───────────────╮
│Case│ALCOHOL (C)│SELF-ESTEEM (SE)│BLOOD ALCOHOL CONTENT (BAC)│SELF-DISCLOSURE│
├────┼───────────┼────────────────┼───────────────────────────┼───────────────┤
│1   │        .00│            3.10│                       4.20│           3.70│
│2   │        .00│            2.50│                       1.50│           5.70│
│3   │        .00│            3.00│                       2.00│           4.40│
│4   │        .00│            2.00│                       2.20│           3.20│
│5   │        .00│            3.20│                       1.00│           6.90│
│6   │        .00│            1.80│                       1.70│           3.50│
│7   │        .00│            2.30│                       2.70│           6.40│
│8   │        .00│            2.40│                       2.60│           5.80│
│9   │        .00│            1.00│                       2.80│           4.40│
│10  │        .00│            3.20│                       3.00│           5.80│
│11  │        .00│            1.00│                       3.70│           3.80│
│12  │        .00│            4.70│                       3.30│           5.60│
│13  │        .00│            2.70│                       2.90│           5.00│
│14  │        .00│            4.70│                       3.70│           7.00│
│15  │        .00│            1.80│                       2.30│           4.00│
│16  │        .00│            2.20│                       3.50│           3.60│
│17  │        .00│            3.60│                       2.20│           5.90│
│18  │        .00│            3.90│                       3.40│           5.00│
│19  │        .00│            3.70│                       1.40│           5.70│
│20  │        .00│            2.90│                       2.90│           3.40│
│21  │       1.00│            2.40│                       5.30│           5.30│
│22  │       1.00│            1.70│                       5.40│           5.90│
│23  │       1.00│            4.10│                       6.20│           5.10│
│24  │       1.00│            3.60│                       4.50│           5.90│
│25  │       1.00│            5.00│                       6.90│           5.00│
│26  │       1.00│            2.70│                       2.30│           6.90│
│27  │       1.00│            3.60│                       5.60│           6.30│
│28  │       1.00│            2.60│                       5.50│           3.90│
│29  │       1.00│            3.50│                       4.50│           5.90│
│30  │       1.00│            3.30│                       3.00│           5.90│
│31  │       1.00│            3.40│                       5.10│           4.80│
│32  │       1.00│            2.80│                       4.00│           4.80│
│33  │       1.00│            2.10│                       4.60│           5.80│
│34  │       1.00│            3.30│                       6.70│           6.20│
│35  │       1.00│            3.30│                       4.90│           7.50│
│36  │       1.00│            3.00│                       5.50│           5.50│
│37  │       1.00│            3.50│                       4.60│           7.00│
│38  │       1.00│            4.50│                       5.70│           6.40│
│39  │       1.00│            2.60│                       6.20│           6.00│
│40  │       1.00│            4.60│                       5.10│           4.50│
╰────┴───────────┴────────────────┴───────────────────────────┴───────────────╯
